src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends AbstractAreaCalculator {
+    private int radius;
+    private String color;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Circle - Color: " + color + ", Radius: "
```

Comment:
```suggestion
        System.out.println("Circle - Color: " + getColor() + ", Radius: "
```

remove field color from this class - it is in AbstractFigure